core/src/blocktree.rs

```diff
@@ -1107,13 +1243,7 @@ impl Blocktree {
             data_end_idx
         );
 
-        self.write_blobs(recovered_data)?;
```

Comment:
is this the code that was returning an error before?

Reply:
Sometimes the `Session::reconstruct_blobs` would error because it was called with too few blobs, or it would be called  on a full set, which gave the zeroes.